tools/buildsys/src/args.rs

```diff
@@ -183,6 +204,7 @@ pub(crate) fn rerun_for_envs(build_type: BuildType) {
 #[repr(u8)]
 #[derive(Debug, Clone, Copy)]
 enum BuildFlags {
+    Repack = 0b00000000,
```

Comment:
At least one of the bits should be set to determine which build type it is
```suggestion
    Repack = 0b00001000,
```

Reply:
I was hoping it might get treated as nothing, but it seems to have had the opposite effect. The good news is by switching to `0b00001000` I'm no longer seeing the extraneous cargo directives for the sensitive env vars that I was seeing before.